index.bs

```diff
@@ -124,20 +124,26 @@ spec:fetch; type:dfn; text:value
 <section>
   <h2 id="framwork">Framework</h2>
   <section>
-    <h3 id="features">Features</h3>
-    <p>A <dfn>feature</dfn> controlled by this framework is an API or behaviour
-    which can be enabled or disabled in a document or web worker by referring to
-    it in a <a>feature policy</a>.
-    <p><a>Features</a> have a <dfn>feature name</dfn> keyword, which is a token
-    used in <a>policy directives</a>, and a <a>default allowlist</a>, which
-    defines whether the <a>feature</a> is available to top-level documents, and
-    how access to that feature is inherited by cross-origin frames.</p>
+    <h3 id="features">Policy-controlled Features</h3>
+    <p>A <dfn data-lt="feature|policy-controlled feature">policy-controlled feature</dfn>
```

Comment:
Add `export` attribute. But I think "feature" shouldn't be exported. You can use `<a lt="policy-controlled feature">feature</a>` when linking instead.